Plugins/PrepareMetalShaders/main.swift

```diff
@@ -30,6 +30,20 @@ struct PrepareMetalShaders: BuildToolPlugin {
     /// pattern to rewrite
     private let include = try! Regex("#include \"mlx/backend/metal/kernels/([^\"]*)\"")
 
+    // see Source/Cmlx/mlx/mlx/backend/metal/kernels/CMakeLists.txt
+    let kernels: Set = [
+        "arg_reduce.metal",
+        "conv.metal",
+        "fft.metal",
+        "gemv.metal",
+        "quantized.metal",
```

Comment:
I think `quantized` and `fit` are moved to the JIT. Also what about `gemv_masked`, that should be here no?

Reply:
Yes, correct on quantized and fft -- those are moved to JIT in 0.15.2.  Good catch!

Checking on gemv_masked

Reply:
Added gemv_masked